### ISSUE TITLE
[BUG] onnx-simplifier==0.4.25 cannot do shape inference in some onnx

### ISSUE THREAD

**Issue by TangChangcheng** [daquexian/onnx-simplifier#322]
**Describe the bug**

The onnx model does have input shape, but after simplification its input shape becomes [unk_1, ...], even I set `--overwrite-input-shape xxx` and it does not work.

**Model**
To reproduce the problem, please post download link of your model here, or send your model to daquexian566@gmail.com
[AvgPool_chw_2x4093x3126_k_2x12_p_0x0x0x0_s_2x11.onnx.zip](https://github.com/daquexian/onnx-simplifier/files/14428369/AvgPool_chw_2x4093x3126_k_2x12_p_0x0x0x0_s_2x11.onnx.zip)

**Comment by TangChangcheng** [comment]
This problem is solved when I degrade onnxsim to 0.4.13, but I still want to ensure why it happens in 0.4.25

**Comment by congyang12345** [comment]
I have the same problem, you can try 
```
pip install onnxslim

onnxslim onnx_model slimmed_onnx_model
```
and the shape can be inferred.

**Comment by daquexian** [comment]
Thanks for your issue! Fixed in v0.4.36. Please update onnxsim by `pip install -U onnxsim`
